lib/board.rb

```diff
@@ -0,0 +1,54 @@
+class Board
+  require 'pry'
+  attr_accessor :cells, :board
+
+  def initialize
+    @board = Array.new(9, " ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.